source/math/FloatVectorList.ooc

```diff
@@ -247,15 +235,97 @@ FloatVectorList: class extends VectorList<Float> {
 		tempVector free()
 		result
 	}
+	/* calculate median without copying and sorting the vector
+			WARNING: this function can partially rearange the elements in vector
+	*/
+	fastMedian: func (start := 0, end := -1) -> Float {
+		if (end < 0)
+			end := this count - 1
```

Comment:
This should be `end = this count - 1` - there's no need to change `end`'s type, and it results in a warning that `end` is unused. Anyway, why would you even get the median for the range (`x`..`y`) where `y` is a negative number?